devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -225,10 +226,12 @@ class HumanStateProvider::impl
 
     // Secondary calibration
     std::unordered_map<std::string, iDynTree::Rotation> secondaryCalibrationRotations;
+    std::unordered_map<std::string, iDynTree::Position> secondaryCalibrationPositions;
+    iDynTree::Transform secondaryCalibrationWorld;
     void eraseSecondaryCalibration(const std::string& linkName);
     void selectChainJointsAndLinksForSecondaryCalibration(const std::string& linkName, const std::string& childLinkName,
                                                   std::vector<iDynTree::JointIndex>& jointZeroIndices, std::vector<iDynTree::LinkIndex>& linkToCalibrateIndices);
-    void computeSecondaryCalibrationRotationsForChain(const std::vector<iDynTree::JointIndex>& jointZeroIndices, const iDynTree::Transform &baseTransform, const std::vector<iDynTree::LinkIndex>& linkToCalibrateIndices);
+    void computeSecondaryCalibrationRotationsForChain(const std::vector<iDynTree::JointIndex>& jointZeroIndices, const iDynTree::Transform &baseTransform, const std::vector<iDynTree::LinkIndex>& linkToCalibrateIndices, const std::string& baseLinkName);
```

Comment:
A curiosity, would it be more meaningful to rename `baseLinkName` to `refLinkForCalibration` ? I suppose this base link is not the same as the base link of the `kindyncomputations` ? We are not actually changing the default base link of the model with this function call right? or did I miss something?

Reply:
Yes, you are right. There is actually no reason for referring at it as baseLink, and it is, in fact, a source of confusion (e.g. https://github.com/robotology/human-dynamics-estimation/pull/237#discussion_r572866358).

I will change it accordingly to your suggestion.

Reply:
Done https://github.com/robotology/human-dynamics-estimation/pull/237/commits/a7f8724e968dfc7e5adc5b14a15bbd6d81735348